request.py

```diff
@@ -21,8 +22,28 @@ class Auth():
     LOGIN_URL="https://account.xiaomi.com/pass/serviceLogin?sid={}&_json=true&passive=true&hidden=false"
     START="&&&START&&&"
     def login_tui(self, sid):
-        self.login(input("Please visit {} and copypaste the data from your web browser here: ".format(self.LOGIN_URL.format(sid))))
-    def login(self, data):
+        self.login()
+    def login(self):
+        session = requests.Session()
+        session.get("https://account.xiaomi.com/pass/serviceLogin?sid=passport&json=false&passive=true&hidden=false&_snsDefault=facebook&_locale=en")
+        session.get("https://account.xiaomi.com/pass/js/info?type=notice&_locale=en")
+        us = input("Insert xiaomi account user (if you have concern on how it's used please read the source code): ")
+        ps = hashlib.md5(input("Insert xiaomi account password (same as above): ").encode("utf-8")).hexdigest().upper()
+        loginData = {
+            '_json': 'true',
+            'callback':     'https://account.xiaomi.com',
+            'sid':          'passport',
+            'qs':           '%3Fsid%3Dpassport%26json%3Dfalse%26passive%3Dtrue%26hidden%3Dfalse%26_snsDefault%3Dfacebook%26_locale%3Den',
+            '_sign':        '2&V1_passport&O3CI2mWi6BiCSNAR7hRK9CatpIw=',
+            'serviceParam': "{'checkSafePhone':'false'}",
+            'user':         us,
+            'hash':         ps,
+            'cc':           '+39',
```

Comment:
Well, I forgot to do It. It could be set ti blank even tho I think that if someone tries ti login with phone number then his phone number must go here

Reply:
Okay, can you remove please